### DISCUSSION TITLE
.gitignore Question

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#22842] [New to GitHub]
<p>Can someone confirm that filenames in <code>.gitignore</code> are matched in any directory?</p>
<p>Ex: If I have an entry <code>fred.txt</code> that’ll match:</p>
<ul>
<li><code>/fred.txt</code></li>
<li><code>/org/someco/whateverpackage/fred.txt</code></li>
<li>etc</li>
</ul>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/kingthorin">@kingthorin</a>,</p>
<p>Yes, files with the same name and extension will be ignored, even in other directories.</p>

**Comment by airtower-luna** [comment]
<p>A little additional detail that may be useful sometimes: It will match in subdirectories. So patterns in <code>/.gitignore</code> will match anywhere in the repository, but if you have e.g. an <code>/org/.gitignore</code> the patterns in there will only match in the <code>/org</code> directory.</p>
